app/format/format.tsx

```diff
@@ -327,7 +327,10 @@ export function formatRole(role: string): string | null {
   return null;
 }
 
-export function formatWithCommas(num: number | Long) {
+export function formatWithCommas(num: number | Long | Number | undefined) {
+  if (!num) {
```

Comment:
```suggestion
  if (num === undefined || num === null) {
```

(otherwise returns `""` when `num === 0`)

Reply:
Good catch!